README.md

```diff
@@ -98,3 +98,9 @@ If you're looking to raise an issue with this module, please create a new issue
 | <a name="output_maintenance-window-target-id"></a> [maintenance-window-target-id](#output\_maintenance-window-target-id) | The target id for the maintenance window |
 | <a name="output_patch-resource-group-arn"></a> [patch-resource-group-arn](#output\_patch-resource-group-arn) | The resource group arn for patching |
 <!-- END_TF_DOCS -->
+
+## Issues
```

Comment:
Why not add this directly into the repository issues, instead of through the readme?